src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -306,18 +323,31 @@ Task AbandonAsyncOnCancellation(BrokeredMessage message)
             {
                 logger.InfoFormat("Abandoning brokered message {0}", message.MessageId);
 
-                await message.SafeAbandonAsync(propertiesToModify).ConfigureAwait(false);
+                if (await message.SafeAbandonAsync(propertiesToModify).ConfigureAwait(false))
+                {
+                    logger.InfoFormat("Brokered message {0} abandoned successfully.", message.MessageId);
+                }
+                else
+                {
+                    logger.InfoFormat("Abandoning brokered message {0} failed. Message will reappear after peek lock duration is over.", message.MessageId);
```

Comment:
Both logs shouldn't be `Warn` or `Info`, but `Debug`. There's nothing that user can do about this and we don't wan't to use `Info` because it will pollute the log file for no reason.